package.json

```diff
@@ -110,13 +111,15 @@
     "lint-staged": "^12.1.2",
     "load-grunt-tasks": "^5.1.0",
     "npm-run-all": "^4.1.1",
-    "octonode": "^0.9.4",
+    "octonode": "^0.10.2",
+    "postcss": "^7.0.39",
     "postcss-clean": "^1.2.2",
     "prettier": "1.16.4",
-    "release-it": "^14.11.8",
+    "release-it": "^14.14.3",
     "s3": "^4.4.0",
     "semver": "^5.7.1",
-    "simple-git": "^3.7.1"
+    "simple-git": "^3.7.1",
+    "underscore": "^1.13.3"
```

Comment:
Why we need undersocre as dev-dep now?

Reply:
I am not sure, but the answer lies in the ways of npm. The resolution entry of unbderscore did not do its job until I installed it as a direct (dev) dependency.

Reply:
I will try to uninstall it again it and see if things still work

Reply:
If the checks also succeed, it does work without underscore as a direct dependency 👍